docs/getting-started-guides/minikube.md

```diff
@@ -99,6 +99,14 @@ Kubectl is now configured to use the cluster.
 
 This will build and start a lightweight local cluster, consisting of a master, etcd, Docker and a single node.
 
+If you are behind a web proxy, you will need to pass this information in e.g. via
```

Comment:
"in e.g. via" is a strange phrasing. 

Perhaps removing this phrase and just leaving it as "you need to pass this information to minikube, following the example below:"